internal/pkg/cli/deploy/workload.go

```diff
@@ -275,6 +295,7 @@ func newWorkloadDeployer(in *WorkloadDeployerInput) (*workloadDeployer, error) {
 		envSess:                  envSession,
 		store:                    store,
 		envConfig:                envConfig,
+		labeledTermPrinter:       labeledTermPrinter,
```

Comment:
```suggestion
		labeledTermPrinter:       syncbuffer.NewLabeledTermPrinter,
```

Reply:
This is giving me an error of `IncompatibleAssign` 🤔 . if I directly use syncbuffer.NewLabeledTermPrinter  because the the function signature of the labeledTermPrinter field in workloadDeployer struct does not match the signature of syncbuffer.NewLabeledTermPrinter function.

Reply:
ah gotcha, it's bc the signature expects the _interface_ to be returned 👍